pennylane_lightning/core/lightning_newAPI_base.py

```diff
@@ -35,7 +35,7 @@
 
 @simulator_tracking
 @single_tape_support
-class LightningBase(ABC, Device):
```

Comment:
If inheritance order matters we probably have methods with the same name in both parents. Is this the case? Do you happen to know which?

Reply:
I checked and `Device` has the parent `ABC` too. In this case, I will remove the parent class `ABC` from the `LightningBase`. Thank you @AmintorDusko for the catch :rocket: